docs/source/index.rst

```diff
@@ -0,0 +1,200 @@
+venv-update: quick, exact
+=========================
+`Issues <https://github.com/yelp/pip-faster/issues>`_ |
+`Github <https://github.com/yelp/pip-faster>`_ |
+`CI <https://circleci.com/gh/Yelp/pip-faster>`_ |
+`PyPI <https://pypi.python.org/pypi/pip-faster/>`_
+
+Release v\ |release| (:ref:`Installation`)
+
+.. toctree::
+   :hidden:
+
+   Documentation overview <self>
+   venv-update
+   pip-faster
+
+Introduction
+------------
+
+venv-update is an `MIT Licensed`_ tool to quickly and exactly update
+a large python project's requirements.  
```

Comment:
I think this is somewhat ambiguous, what does "update a large project's requirements" mean? It might sound like you're actually updating versions rather than just syncing a virtualenv with a list of requirements.txt-style files.